linker_scripts/object_script.txt

```diff
@@ -91,6 +91,15 @@ D_06009120 = 0x06009120;
 D_06005458 = 0x06005458;
 D_0600788C = 0x0600788C;
 
+/* en_ginko_man */
```

Comment:
FYI names for object segments are not required, but symbols are. Names are accepted, however.